okapi/static/ocgpx/1/ocgpx.xsd

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xsd:schema
+    xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+    xmlns="https://opencaching.pl/okapi/static/ocgpx/1"
+    targetNamespace="https://opencaching.pl/okapi/static/ocgpx/1"
+    elementFormDefault="qualified">
+
+    <xsd:annotation>
+        <xsd:documentation>
+            Common Opencaching extensions to the Topografix GPX schema.
+        </xsd:documentation>
+    </xsd:annotation>
+
+    <xsd:element name="cache" type="cache_type" />
```

Comment:
If we're only trying to add attributes to an existing `groundspeak:geocache` element, then I guess we shouldn't declare our own `cache` element. Instead, we should declare only specific elements and annotate them properly (e.g. "this element can be put only as the child of the `groundspeak:geocache` element").

Similar is true for the `log` element below.

Reply:
Agree for the logs. They depend on groundspeak, because logs are added to GPX by the groundspeak extension.

`oc:cache` (as proposed) is independent from `groundspeak:cache`; it's directly attached to the `wpt` element, which represents a geocache (it has `<sym>Geocache</sym>`). There will probably be more Opencaching-specific cache attributes be added, so it may make sense to group them as `oc:cache`.

Reply:
Okay, sounds reasonable. `wpt` can also represent "additional waypoint", but we may use a different element for extending this one.